ansible/roles/kubernetes/tasks/debian.yml

```diff
@@ -97,6 +97,19 @@
 # sysctl net.bridge.bridge-nf-call-iptables=1 to pass bridged IPv4 traffic to iptables’ chains.
 # This is a requirement for some CNI plugins to work.
 # https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#letting-iptables-see-bridged-traffic
+- name: Load br_netfilter
+  modprobe:
+    name: br_netfilter
+    state: present
+
+- name: persist br_netfilter load through reboots
+  lineinfile:
+    path: '{{ os_modules_load_list }}'
+    state: present
+    regex: '^br_netfilter'
+    line: 'br_netfilter'
+  notify: reboot hosts
```

Comment:
This is not needed as the modprobe already takes care of loading the module within the kernel.

Reply:
I was in the middle of typing this as the email came in. :+1: